### PR TITLE
PLAT-11815 Django Scaffold Update (Scaffold)

### PR DESCRIPTION
**Pull request by mabdullahabid**

### CODE REVIEW

**Review by Hesbon5600 — APPROVED** [review]

**Comment by driverdan** [comment]
@mabdullahabid Please fix the conflicts

**Comment by mabdullahabid** [comment]
> @mabdullahabid Please fix the conflicts

Done

**Comment by mabdullahabid** [comment]
@driverdan please merge

**Comment by driverdan** [comment]
@mabdullahabid This needs a re-review since you made changes.

File: {{cookiecutter.project_slug}}/Pipfile [hunk 1]
```diff
@@ -10,8 +10,8 @@ python_version = "3.8"
 
 [packages]
 cryptography = "~=40.0.2"
-django = "~=2.2.26"
-"boto3" = "~=1.28.25"
+django = "==3.2.20"
+"boto3" = "~=1.28.1"
```
Comment (sgmagar) [inline {{cookiecutter.project_slug}}/Pipfile:14]: Is this downgrade of boto3 required?

**Review by sgmagar — COMMENTED** [review]
minonr comment

File: {{cookiecutter.project_slug}}/Pipfile (hunk 1 above)
Comment (mabdullahabid) [reply {{cookiecutter.project_slug}}/Pipfile]: Fixed.

**Review by cruz636 — APPROVED** [review]

**Pull request merged**